Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml

```diff
@@ -0,0 +1,155 @@
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Cisco Meraki Documentation
+  Link: https://documentation.meraki.com/ 
+Description: |
+  This ASIM parser supports normalizing Cisco Meraki logs to the ASIM Authentication normalized schema. Cisco Meraki events are generated from network activity and security events from Meraki devices such as firewalls, switches, and access points. These logs are captured through the Cisco Meraki Sentinel connector which uses a Linux agent to collect logs in Syslog format.
+ParserName: ASimAuthenticationCiscoMeraki
+EquivalentBuiltInParser: _ASim_Authentication_CiscoMeraki
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+
+ParserQuery: |
+  let EventResultDetailsLookup = datatable (reason: string, EventResultDetails: string)
+      [
+      "0", "Other",
+      "1", "Other",
+      "2", "Password expired",
+      "3", "Other",
+      "4", "Session expired",
+      "5", "Other",
+      "6", "Other",
+      "7", "Other",
+      "8", "Other",
+      "9", "Other",
+      "10", "Logon violates policy",
+      "11", "Logon violates policy",
+      "12", "Other",
+      "13", "Logon violates policy",
+      "14", "Other",
+      "15", "Other",
+      "16", "Other",
+      "17", "Other",
+      "18", "Incorrect key",
+      "19", "Incorrect key",
+      "20", "Incorrect key",
+      "21", "Other",
+      "22", "Other",
+      "23", "Other",
+      "24", "Logon violates policy",
+  ];
+  let EventFieldsLookup = datatable (
+      LogSubtype: string,
+      EventResult: string,
+      EventType: string,
+      EventSeverity: string
+  )
+      [
+      "8021x_auth", "Success", "Logon", "Informational",
+      "wpa_auth", "Success", "Logon", "Informational",
+      "splash_auth", "Success", "Logon", "Informational",
+      "8021x_eap_success", "Success", "Logon", "Informational",
+      "8021x_deauth", "Success", "Logoff", "Informational",
+      "8021x_client_deauth", "Success", "Logoff", "Informational",
+      "wpa_deauth", "Success", "Logoff", "Informational",
+      "8021x_eap_failure", "Failure", "Logon", "Low",
+      "disassociation", "Failure", "Logon", "Low",
+  ];
+  let parser = (disabled: bool=false) {
+      union isfuzzy=true
+          (
+          meraki_CL
+          | project-rename LogMessage =  Message
+          ),
+          (
+          Syslog
+          | where Computer in (_ASIM_GetSourceBySourceType('CiscoMeraki'))
+          | project-rename LogMessage =  SyslogMessage
+          )
+      | where not(disabled)
+          and LogMessage has "events"
+          and (LogMessage has_any ("8021x_auth", "wpa_auth", "splash_auth", "8021x_deauth", "8021x_client_deauth", "wpa_deauth", "8021x_eap_failure", "8021x_eap_success") or (LogMessage has "disassociation" and LogMessage has "auth_neg_failed"))
```

Comment:
can we put his in a variable?
8021x_auth", "wpa_auth", "splash_auth", "8021x_deauth", "8021x_client_deauth", "wpa_deauth", "8021x_eap_failure", "8021x_eap_success

Reply:
Done